docs/resources/azure_redis_caches.md

```diff
@@ -0,0 +1,118 @@
+---
+title: About the azure_redis_caches Resource
+platform: azure
+---
+
+# azure_redis_caches
+
+Use the `azure_redis_caches` InSpec audit resource to test properties related to Azure Redis Cache for a resource group or the entire subscription.
+
+## Azure REST API version, endpoint and http client parameters
+
+This resource interacts with api versions supported by the resource provider.
+The `api_version` can be defined as a resource parameter.
+If not provided, the latest version will be used.
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the http client will be used.
+For more information, refer to the resource pack [README](../../README.md).
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure).
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+## Syntax
+
+An `azure_redis_caches` resource block returns all Azure Redis Caches within a Resource Group.
+```ruby
+describe azure_redis_caches(resource_group: 'my-rg') do
+  #...
+end
+```
+or
+```ruby
+describe azure_redis_caches(resource_group: 'my-rg') do
+  #...
+end
+```
+## Parameters
+
+The parameter should be provided for valid query
+- `resource_group`
+
+| Name                            | Description                                                                      |
+|---------------------------------|----------------------------------------------------------------------------------|
+| resource_group                  | Azure resource group that the targeted resource resides in. `MyResourceGroup`    |
```

Comment:
Ian - where to find the relevant microsoft doc for this table?

Reply:
these parameters are defined for an inspec resource .. and you can't find a relevant doc at microsoft..